legacy-app/src/app/modules/event-templates/components/event-form-dialog/event-form-dialog.component.ts

```diff
@@ -35,12 +35,12 @@ export class EventFormDialogComponent {
     this.dialogForm = this.fb.group({
       title: ['', Validators.required],
       icon: ['', Validators.required],
-      description: ['', Validators.required],
-      comment: ['', Validators.required],
+      description: ['TBD', Validators.required],
```

Comment:
I think we should consider not allowing to create an event until these are filled. It is important to cut down on the empty templates.

Reply:
No, usually Admins create an empty template for the responsible tutor to fill out